### ISSUE TITLE
[Bug]: Fatal crash after latest version was installed

### ISSUE THREAD

**Issue by skcuslleb** [advplyr/audiobookshelf#3567]
### What happened?

I see crash in the logs when I try to play my audio book through Plappa (which worked fine before yesterdays updated.

### What did you expect to happen?

No crash

### Steps to reproduce the issue

1. Start the server
2. Attempt to play the audio book through Plappa

### Audiobookshelf version

v2.16.1

### How are you running audiobookshelf?

Docker

### What OS is your Audiobookshelf server hosted from?

Other (list in "Additional Notes" box)

### If the issue is being seen in the UI, what browsers are you seeing the problem on?

Other (list in "Additional Notes" box)

### Logs

```shell
[2024-10-29 14:11:03.559] ERROR: [LibraryItemController] Failed to download file "<****REDACTED****>.m4b" Error: Request aborted
    at onaborted (/node_modules/express/lib/response.js:1052:15)
    at Immediate.<anonymous> (/node_modules/express/lib/response.js:1094:9)
    at process.processImmediate (node:internal/timers:483:21) {
  code: 'ECONNABORTED'
}
[2024-10-29 14:11:03.610] FATAL: [Server] Unhandled rejection: TypeError: Receiver must be an instance of class LibraryItemController
    at ApiRouter.downloadLibraryFile (/server/controllers/LibraryItemController.js:868:12) 
promise: Promise {
  <rejected> TypeError: Receiver must be an instance of class LibraryItemController
      at ApiRouter.downloadLibraryFile (/server/controllers/LibraryItemController.js:868:12)
}
```

### Additional Notes

Docker running on Synology. Audio book client is Plappa on iOS

**Comment by averageyogi** [comment]
Mentioned here https://github.com/advplyr/audiobookshelf/pull/3553#issuecomment-2445031131 and brought up in Plappa repo

**Comment by nichwall** [comment]
This has been fixed in 2.16.2, released about an hour ago

**Comment by skcuslleb** [comment]
Fix confirmed in 2.16.2
